Subhajit/Robot_Framework/Resources/Goibibo_Keywords.robot

```diff
@@ -0,0 +1,25 @@
+*** Settings ***
+Library     SeleniumLibrary
+
+*** Keywords ***
+User nevigate to Goibibo website
+    open browser    https://www.goibibo.com/        chrome
```

Comment:
This URL `https://www.goibibo.com/` can be different for different env's, So we should parametrize and parse it from configuration Env related files